Assets/Scripts/BotController.cs

```diff
@@ -34,7 +36,10 @@ public class BotController : MonoBehaviour
     // Start is called before the first frame update
     void Start()
     {
-        
+        foreach (BodyPart bodyPart in bodyParts)
```

Comment:
I'm not sure this makes sense unless we remove the AddHitPoints from the BodyParts themselves.